index.html

```diff
@@ -17,24 +17,188 @@
+           
+           <ul class="languages">
+             <li class="langn">html</li>
+             <li class="langn">bootstrap</li>
+             <li class="langn" >Ruby</li>
+           </ul>
+           <button class="project-button fullwidth-button">See Project</button>
+       </div>
+       </section>
+
+         <!-- project 5-->
+      <section class="projects" id ="pro-5">
+         <div class = "project-data">
+           <h3 class="project-title">Professional Art Printing Data</h4>
+           <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+           
+           <ul class="languages">
+             <li class="langn">html</li>
+             <li class="langn">bootstrap</li>
+             <li class="langn" >Ruby</li>
+           </ul>
+           <button class="project-button fullwidth-button">See Project</button>
+       </div>
+       </section>
+
+         <!-- project 6-->
+      <section class="projects" id ="pro-6">
+         <div class = "project-data">
+           <h3 class="project-title">Professional Art Printing Data</h4>
+           <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+           
+           <ul class="languages">
+             <li class="langn">html</li>
+             <li class="langn">bootstrap</li>
+             <li class="langn" >Ruby</li>
+           </ul>
+           <button class="project-button fullwidth-button">See Project</button>
+       </div>
+       </section>
+    </section>
+
+    <section class="about-me">
+      <div class="about-me-main">
+      <h2 class="section-header">About me</h2>
+      <p class="about-me-description">
+        Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <!-- <img src="images/illustration-about-me-mobile@2x.svg" class="about-illus"> -->
+      <button class="project-button-get-resume">Get My Resume</button>
+      </div>
+    <hr>  
+    <ul class = "skill-list">
+      <li>
+        <div class="lang-wrapper">
+          <img src="images/Rectangle 55.svg"  class="Rectangle">
+          <h3 class="language">Language</h3>
+        </div>
+        <ul class="language-list">
+          <li>Javascript</li> 
+          <li>Ruby</li>
+          <li>Html</li>
+          <li>Css</li>
+        </ul>
+      </li>
+      <li>
```

Comment:
The list of icons should be grouped by a `ul` tag and each icon should be listed an `li` tag and a `link` (a tag). There is no need to include a div and header tag in each of the list. To make it easier, you can consider using a nested list ie. a list within a list

Reply:
can you brief it please

Reply:
> can you brief it please

Kindly check out [this link](https://www.webucator.com/article/how-to-properly-nest-lists-in-html/) for more information on that